include/deal.II/grid/reference_cell.h

```diff
@@ -1404,6 +1404,8 @@ ReferenceCell::face_to_cell_lines(const unsigned int  face,
   AssertIndexRange(face, n_faces());
   AssertIndexRange(line, face_reference_cell(face).n_lines());
 
+  static constexpr unsigned int X = numbers::invalid_unsigned_int;
```

Comment:
Maybe add a short comment that this gets used for nicer formatting of the tables for pyramids and wedges further down?

Reply:
This is already done all over the place in that file!

Reply:
OK, then nevermind.